index.js

```diff
@@ -97,20 +116,32 @@ function resolver(src) {
         }
 
         resolve(promise);
+        head.appendChild(element);
       }
 
-      head.appendChild(element);
     });
 }
 
+var CSS_REGEXP = /\.css$/;
+var JS_REGEXP = /\.js$/;
+
 function normalizeSource(src) {
   if (isPlainObject(src)) {
     return src;
   }
 
+  var type;
+  if (CSS_REGEXP.test(src)) {
+    type = 'style';
+  } else if (JS_REGEXP.test(src)) {
+    type = 'script';
+  } else {
+    type = 'xhr';
```

Comment:
End of day brain fart is why
For some reason I was treating `type` to mean the transport type, as if we use the style element to load css remotely...